backend/attachRenderer.js

```diff
@@ -50,7 +50,9 @@ function attachRenderer(hook: Hook, rid: string, renderer: ReactRenderer): Helpe
     extras.getNativeFromReactElement = function(component) {
       try {
         return renderer.Mount.getNode(component._rootNodeID);
-      } catch (e) {}
+      } catch (e) {
+        return undefined;
+      }
```

Comment:
Yea, `consistent-return` ESLint rule doesn’t like this. The idea is you either `return;` everywhere in a function or `return <something>;` in it.